app/routers/v1/admin/OrganizationRouter.js

```diff
@@ -0,0 +1,14 @@
+// OrganizationRouter
+module.exports = exports = {
+  '/organization' : {
```

Comment:
Now that I think about it, you should be allowed to just use any of the routes under /v1/admin/organizations not just post and delete.  Keep the resource portion and that will be sufficient.